.travis.yml

```diff
@@ -43,6 +43,9 @@ env:
     - "GEM=ar:postgresql"
     - "GEM=aj:integration"
     - "GEM=guides"
+  global:
+  - secure: ctwuhhY76OFhI/lmK054lL4Tzmney3l3gpnCkzlZ8idMAbeKAPOGdJxx7QUVQi3dY3N9lg+iAhKGNk+/J/ze7SG0gjGFsjAjqzIZonirwkfNIZrTgWmkclE4IbqAU9rG7kvsobzlMNNpS7sfmoT4vabvsJyCMObuEfBnaVZE0DA=
+  - secure: bcEdBn7qw0KpbPIzTMN0fL6CNmpUZHGm3QJfThRzDm9S4X1Y5Yo063TXtb8nvy4P7019MPGezNd6Xw3kslB1D3g6R9nnEUrB0jd94VdKqw+HJzHkaT/hdGoB5ODVgIfrUcfAuElwT+7jd1Fx5JjtYbEgQjlVOyw0mk4SiV88IJQ=
```

Comment:
FYI, these env vars won't be set when testing pull requests: https://docs.travis-ci.com/user/pull-requests#Security-Restrictions-when-testing-Pull-Requests

Since leaking the access token is low risk for free-to-open-source Sauce Labs accounts, Trix just base64s them to avoid showing up in search results 🙈: https://github.com/basecamp/trix/blob/master/bin/ci#L5-L6. This compromise allows PRs to be tested. Inspired by https://github.com/jquery/qunit/commit/0cf737d46775aecb06780e3df36cb9cac6d01b0c.

/cc @jeremy